src/raiden_libs/matrix.py

```diff
@@ -75,17 +74,6 @@ def check_and_count(self, sender: Address, added_bytes: int) -> bool:
         return True
 
 
-def message_from_dict(data: Dict[str, Any]) -> Message:
-    if "_type" not in data:
-        raise SerializationError("Invalid message data. Can not find the data type") from None
-    if data["_type"] not in VALID_MESSAGE_TYPES:
```

Comment:
Not sure if we should check somewhere that the message is one of the types in `SERVICE_MESSAGES`

Reply:
Messages of unexpected types will just not be handled in `service.handle_message`. I don't see a need for checks in addition to that. This would mean we can remove `SERVICE_MESSAGES` and `VALID_MESSAGE_TYPES`.